packages/frontend/src/components/ChartConfigPanel/FieldLayoutOptions.tsx

```diff
@@ -86,6 +83,10 @@ const FieldLayoutOptions: FC<Props> = ({ items }) => {
         return items.filter((item) => isDimension(item));
     }, [items]);
 
+    const chartInvolvesMetrics = resultsData
+        ? resultsData.metricQuery.metrics.length > 0
```

Comment:
Instead of checking the metric query we should check if the fields being used in the x and y axis are dimensions. 
Eg: The metric query has dimension A, B and C and metric Q and P. The chart uses in the x axis dimension A and in the y axis dimension B. Since the chart only uses dimension A and B the group input should be disabled.

Reply:
The group should be enabled if even there is custom metric in one axis ?

Reply:
Correct, group should be enabled if the y axis is using a metric, custom metric or table calculation